terraform/deployments/ecr/main.tf

```diff
@@ -209,6 +209,42 @@ resource "aws_iam_user_policy" "github_ecr_user_policy" {
   })
 }
 
+resource "aws_ecr_lifecycle_policy" "ecr_lifecycle_policy" {
+  for_each   = toset([for repo in local.repositories : aws_ecr_repository.repositories[repo].name])
+  repository = each.key
+
+  policy = <<EOF
```

Comment:
Tip for next time: use `jsonencode()` instead of string templates, so that the parser can check the syntax ahead of time.